CMakeLists.txt

```diff
@@ -98,6 +98,7 @@ aws_prepare_symbol_visibility_args(${PROJECT_NAME} "AWS_CAL")
 aws_add_sanitizers(${PROJECT_NAME} BLACKLIST "sanitizer-blacklist.txt")
 
 aws_use_package(aws-c-common)
+aws_use_package(aws-checksums)
```

Comment:
You also need to add this as an upstream in `builder.json`

Reply:
This will fix CI failures

Reply:
resolved in patch2.